_episodes/01-start-existing-repo.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: episode
+title: In-browser session
+teaching: 25
+exercises: 0
+objectives:
+  - See an existing repository in action.
+  - Browse the history.
+  - See the big picture first before we dive into details.
+---
+
+# In-browser session
+
+- We will explore and visualize an **existing Git repository** on GitHub and make small changes to it.
+- The goal of this episode is not to teach GitHub.
+- We will start out with a jam session on the web (work on existing repository).
+- Then we will practice scales on our local instruments (practice Git on our laptops).
+- Then (tomorrow) we regroup again and play orchestra music together.
+
+---
+
+## Why
+
+- Often our first contact with Git is an existing repository.
+- The existing repository is often on GitHub (but this demonstration applies equally well to GitLab or Bitbucket).
+- See the big picture before we focus.
+- Git commands that we will learn in the following session will make more sense.
+
+---
+
+## Do not worry
+
+- If you are new to Git, many concepts will be new and a bit magic, we will revisit and explain them later.
```

Comment:
- Instead, what about: "If you are new to version control, don't focus on what we do.  Focus on what comes out of it."